build.gradle

```diff
@@ -0,0 +1,54 @@
+apply plugin: 'java'
+
+group = 'com.pi4j'
+version = '0.0.1'
+
+description = """Pi4J :: MINIMAL EXAMPLE :: Sample minimal project"""
+project.buildDir = 'target'
```

Comment:
This deviates from the conventional Gradle directory layout. I would suggest to keep it simple and as Gradle users would expect.
```suggestion
```